src/components/CreateVmWizard/CreateVmWizard.js

```diff
@@ -388,11 +392,15 @@ class CreateVmWizard extends React.Component {
 
   wizardGoToStep (newStepIndex) {
     const { activeStepIndex } = this.state
+    const moveBackward = newStepIndex < activeStepIndex
     const activeStep = this.wizardSteps[activeStepIndex]
     const newStep = this.wizardSteps[newStepIndex]
 
+    // enable leaving the current step in case of moving backward
+    const preventExit = moveBackward ? false : activeStep.preventExit
+
     // make sure we can leave the current step and enter the new step
-    if (activeStep.preventExit || newStep.preventEnter) return
+    if (preventExit || newStep.preventEnter) return
```

Comment:
I'm not sure it makes sense to touch the GoToStep handling in this PR.  It goes along with the back button hanlding that @hstastna was talking about.  All that you're doing here is allowing the back button to be clicked always.

Let's just leave current functionality in place.  If the editing is invalid, force the user to fix and save or cancel the change.  We can handle more complex navigation consistently across steps in future issues (if an issue for it doesn't already exist).

Reply:
If we force the user to fix and save or cancel the change, we should implement the same behavior also in https://github.com/oVirt/ovirt-web-ui/pull/1221 to be consistent.

Reply:
The `this.props.onUpdate({ valid: validAndUniqueName })` work can easily be replicated to #1221 after this PR is merged.

Reply:
With #1260 both this PR and #1221 can use the same `onUpdate` calls.